app/Http/Requests/Trips/StoreRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests\Trips;
+
+use App\Models\User;
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'name' => 'required',
+            'start_date' => [
+                'required',
+                'date',
+            ],
+            'end_date' => [
+                'required',
+                'date',
+                'after_or_equal:start_date',
+            ],
+            'cover_photo' => [
+                'nullable',
+                'file',
+                'mimes:jpg,png',
+                'max:20000',
+            ],
+        ];
+    }
+
+    /**
+     * @todo Think about how we can better handle this even though we're using sanctum so that is added to all routes
+     */
+    public function authorize(): bool
+    {
+        $user = $this->user();
+
+        if (!$user instanceof User) {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
Personally, I think this route should be protected by middleware by default if its a simple auth check, and this method should be omitted unless there is a specific per-user check that is needed.